streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/util/JniUtils.java

```diff
@@ -0,0 +1,58 @@
+package org.ray.streaming.runtime.util;
+
+import com.google.common.base.Strings;
+import java.lang.reflect.Field;
+import org.ray.runtime.RayNativeRuntime;
+import org.ray.runtime.util.FileUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JniUtils {
```

Comment:
Can you just put this function in ray/java/runtime? Since code is the same.

Reply:
done